lib/widgets/element_on_list.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+import '../classes/news.dart';
+import '../Screens/specific_story.dart';
+
+class ElementOnList extends StatefulWidget {
+  Future<News> specificNews;
+  ElementOnList({super.key, required this.specificNews});
+
+  @override
+  State<ElementOnList> createState() => _ElementOnListState();
+}
+
+class _ElementOnListState extends State<ElementOnList> {
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        Navigator.push(
+            context,
+            MaterialPageRoute(
+              builder: (context) => SpecificNews(news: widget.specificNews),
+            ));
```

Comment:
Nit:
```suggestion
          context,
          MaterialPageRoute(
            builder: (context) => SpecificNews(news: widget.specificNews),
          )
        );
```

Reply:
You'll probably need to add a comma before the last `);` to get the formatter to do that.